### ISSUE TITLE
Windows Install

### ISSUE THREAD

**Issue by ardsmedley** [fripon/freeture#36]
Attempting install produces a "CreateProcess failed" error code: 216. Details state that the version of %1 is not compatible with this version of Windows (it's 7 SP1). Which versions should the Windows installer work with? And if it should install on 7, what does the error indicate?

Thanks

**Comment by Asubayo** [comment]
Is it a 32 bit or a 64 bit system ? 
The windows release of FreeTure was expected to run on 64 bits systems.

**Comment by ardsmedley** [comment]
32 bit. Is there anyway to get it to work on 32 bit? Or alternatively would it work on Windows 10 (I think it's 64 bit)?

**Comment by Asubayo** [comment]
No, 32 bits OS can't run 64 bit programs. And there is no 32 bits version of Freeture. 
If you have a computer with windows 10, you can check in the system panel if it is a 64 bits OS version.
If it is the case, the freeture installer should work.

**Comment by ardsmedley** [comment]
Sorry for the delay in getting back to you. Thanks for clarifying that freeture is 64-bit only -- it might be worth renaming the Windows installer as one of the earlier installers is missing a "64" in its name which makes it look like it's 32 bit. (Or perhaps adding a note on the compatibility in the documentation.)

Could you tell me which versions of Windows it should work with: XP / Vista / 7 / 8 / 10?

The other thing we need to bear in mind is that Basler state that Pylon 3.2.x only works up to Windows 7, and as that is a prerequisite for freeture to run there seems to be limited variants of an OS that is compatible.

Thanks for your help.
